File: 백준/Bronze/10989. 수 정렬하기 3/수 정렬하기 3.cc
#include<iostream>

using namespace std;

int main() {
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);
	//freopen_s(new FILE*, "input.txt", "r", stdin);

	int N;
	cin >> N;

	int nums[10001] = { 0 };

	for (int i = 0; i < N; i++) {
		int tmp;
		cin >> tmp;
		nums[tmp]++;
	}

	for (int i = 1; i < 10001; i++) {
		for (int j = 0; j < nums[i]; j++) {
			cout << i << '\n';
		}
	}

}

File: 백준/Bronze/10989. 수 정렬하기 3/수 정렬하기 3.py
from sys import stdin
input = stdin.readline

N = int(input())
s = [0] * 10001

for _ in range(N):
    s[int(input())] += 1

cnt = 0
for i in range(1, 10001):
    for j in range(s[i]):
        print(i)
        cnt += 1
    if cnt == N:
        break
